workflows/example_configs/methods_config.yaml

```diff
@@ -8,26 +8,27 @@ datasets:
   - abc_atlas_wmb_thalamus
   - libd_dlpfc                            # Visium Comparison
   - visium_breast_cancer_SEDR
-#  - stereoseq_developing_Drosophila_embryos_larvae
   - visium_chicken_heart
+#  - stereoseq_developing_Drosophila_embryos_larvae
+#  - "stereoseq_mouse_embryo"
 
 seed: 2023
 
 # Methods to run for the pipeline, modify based on your need
 use_methods:
```

Comment:
Could some of the newly merged methods be included?

Reply:
Done!